packages/monorepo-scripts/src/utils/discord.ts

```diff
@@ -0,0 +1,27 @@
+import { fetchAsync } from '@0x/utils';
+
+import { constants } from '../constants';
+
+import { utils } from './utils';
+
+export const alertDiscord = async (releaseNotes: string): Promise<boolean> => {
```

Comment:
following our conventions, this should be `alertDiscordAsync`. We do have a custom linter rule for enforce this suffix for method names but I don't think it applies to consts

Reply:
ugh, keep forgetting this convention :(